Week-2/Assignment 1/index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Week2_Assignment1</title>
+  </head>
+  <body>
+    <header>
```

Comment:
Try adding the class title-container to the header

Reply:
done